rpc/runtime-api/src/identity.rs

```diff
@@ -40,5 +40,20 @@ sp_api::decl_runtime_apis! {
         ///
         /// This is an aggregate call provided for UX convenience.
         fn get_key_identity_data(acc: AccountId) -> Option<KeyIdentityData<IdentityId>>;
+
+        /// Returns all valid [`IdentityClaim`] of type `CustomerDueDiligence` for the given `target_identity`.
+        ///
+        /// ```ignore
+        /// curl http://localhost:9933 -H "Content-Type: application/json" -d '{
+        ///     "id":1,
+        ///     "jsonrpc":"2.0",
+        ///     "method": "identity_validCDDClaims",
+        ///     "params":[
+        ///         "0x0100000000000000000000000000000000000000000000000000000000000000",
+        ///         null
+        ///     ]
+        ///   }'
+        /// ```
+        fn valid_cdd_claims(target_identity: IdentityId, cdd_checker_leeway: Option<u64>) -> Vec<IdentityClaim>;
```

Comment:
Need to bump the api version to `3` and add `#[api_version(3)]` to this new method.

Reply:
I have bumped the version on the trait as well, otherwise the default version would still be the old one.